README.md

```diff
@@ -32,7 +32,18 @@ with clusters of apps. There will be several different kinds of clusters, such a
    - Local: http://localhost:####
 3. Your new tab should now show the React app.
 
+## Steps to Re-deploy the Java Servlet
+
+- On the root directory, run `mvn appengine:deploy`.
+
+## Steps to Re-deploy the React App
+
+1. In a seperate terminal from the Java Servlet, cd into client.
```

Comment:
We can skip this step. It doesn't need to be in a separate terminal, since the existing terminal useable after deploying java servlet.

And we can also skip `cd` into `client` (more info in the below comments).